VideoLocker/src/main/java/org/edx/mobile/social/SocialLoginDelegate.java

```diff
@@ -211,21 +213,25 @@ public ProfileModel call( ) {
 
                 // do SOCIAL LOGIN first
                 AuthResponse social = null;
+                HashMap<String, CharSequence> descParams = new HashMap<>();
+                String platformName = environment.getConfig().getPlatformName();
+                descParams.put("platform_name", environment.getConfig().getPlatformName());
+                descParams.put("platform_destination", environment.getConfig().getPlatformDestinationName());
```

Comment:
Err, I mean, it is specific to a small number of strings. But if you have a suggestion for a place to put it, I'm happy to put it there.